docs/source/contributing/index.md

```diff
@@ -1,20 +1,80 @@
 # Contributing
 
 We want you to contribute to JupyterHub in ways that are most exciting
-& useful to you. We value documentation, testing, bug reporting & code equally,
-and are glad to have your contributions in whatever form you wish :)
+and useful to you. We value documentation, testing, bug reporting & code equally,
+and are glad to have your contributions in whatever form you wish.
 
-Our [Code of Conduct](https://github.com/jupyter/governance/blob/HEAD/conduct/code_of_conduct.md)
-([reporting guidelines](https://github.com/jupyter/governance/blob/HEAD/conduct/reporting_online.md))
-helps keep our community welcoming to as many people as possible.
+Be sure to first check our [Code of Conduct](https://github.com/jupyter/governance/blob/HEAD/conduct/code_of_conduct.md)
+([reporting guidelines](https://github.com/jupyter/governance/blob/HEAD/conduct/reporting_online.md)), which help keep our community welcoming to as many people as possible.
+
+This section covers information about our community, as well as ways that you can connect and get involved.
+
+## Contributors
+
+Here are the people who keep JupyterHub running and updated.
+
+```{toctree}
+:maxdepth: 1
+
+contributor-list
+```
+
+## Communication channels
```

Comment:
I don't think we should have new headings and separate toctrees for each single document. I think a single toctree is best, and we should make sure to get this information from the headings we get from those documents.

Reply:
Okay. Let me revert to the previous headings and toctree.